### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> Agenda.json]: Luminosity ratio of 'Car' and 'Location' icon is 2.646:1 which is less than 3:1.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8457]
### Target Platforms

Other

### SDK Version

3.2.2303.22003

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25324.1011)
Application: AdaptiveCardVisualizer
Application Version: 3.2.2303.22003
Tool: Color Contrast Analyzer through Accessibility Insights for Windows

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to the 'Agenda.json' through tab key and activate it.
3. Pick the color for foreground and background of 'Car' and 'Location' icon.
4. Observe the issue.

**Actual Result:**
Luminosity ratio of 'Car' and 'Location' icon is 2.646:1 which is less than 3:1.

**Expected Result:**
Luminosity ratio of 'Car' and 'Location' icon should be greater than or equals to 3:1.

**User Impact:**
Person with visual disability will get impacted as user will not be able to see the icon properly and will not be able to undersatnd the purpose which is represented through icon.

**Reference link:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BB19D1152-56E4-44F4-832A-A650AC312C4E%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&share=IQFSEZ2x5Fb0RIMqplCsMSxOAdw8bEbepI0ybgODJQURMgM&cid=dde16c11-21dc-42b4-977d-fb72afda0045

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

<img width="960" alt="MAS1 4 11 Luminosity ratio of location and by car icon is less than required" src="https://user-images.githubusercontent.com/104483005/231081710-51516f20-0806-4439-9443-dfb8f9e3224e.png">

### Card JSON

```JSON
NA
```

**Comment by vaishali1397** [comment]
Issue is repro'ing in the latest test environment will reactivate on 1 sep.
Test environment:
OS: windows11
Application: AdaptiveCard Visualizer
Application Version: 1.2.2306.30005

![image](https://github.com/microsoft/AdaptiveCards/assets/104483005/e693a4d7-0a3a-4924-8f90-3ff7bac605ad)

**Comment by vagpt** [comment]
This issue is working fine, hence closing this issue.

![image](https://github.com/user-attachments/assets/bea16200-6873-4bdb-a7b7-95c8bdf47207)
